pkg/tagger/collectors/workloadmeta_extract.go

```diff
@@ -78,9 +80,10 @@ var (
 	}
 
 	standardDockerLabels = map[string]string{
-		dockerLabelEnv:     tagKeyEnv,
-		dockerLabelVersion: tagKeyVersion,
-		dockerLabelService: tagKeyService,
+		dockerLabelEnv:      tagKeyEnv,
+		dockerLabelVersion:  tagKeyVersion,
+		dockerLabelService:  tagKeyService,
+		dockerLabelRevision: tagKeyGitCommitSha,
```

Comment:
Standard tags in this context mean [Unified Service Tagging](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes). I'm not sure if the commit ID applies here, and would recommend moving it to `lowCardOrchestratorLabels` maybe.

Reply:
definitely +1

Reply:
Let's also rename it `containerRevisionLabel` as it can be found on container runtimes other than Docker [based on this](https://github.com/opencontainers/image-spec/blob/main/annotations.md)

Reply:
Thanks for catching this! I updated the PR to move it in the lowCard list. I also got rid of the constants to be consistent with the rest of the map, let me know if you want me to re-introduce constants for this.

Reply:
no I think keeping it in the low card map makes sense, thanks!